demo/demo-tekton.sh

```diff
@@ -258,8 +292,10 @@ oc process -f "$tekton_setup_dir/05-build-and-deploy.yaml" -p PROJECT_URI="${GIT
            -p NAMESPACE="${app_namespace}" -p APPLICATION_NAME="${app_name}" \
            -n default > /tmp/05-build-and-deploy.yaml.out 2>/tmp/05-build-and-deploy.yaml.err
 oc apply -n "${app_namespace}" -f /tmp/05-build-and-deploy.yaml.out
+# should delete files from /tmp/?
```

Comment:
We do this in the function which is in the trap() that is invoked when the script exits:
```
function cleanup_and_exit() {
```

Reply:
It did not remove neither `/tmp/05-build-and-deploy.yaml.out` or `/tmp/05-build-and-deploy.yaml.err` to me in all my runs. Can you check if it deletes for you?